"""

Q302: The part of your code where you think
 an exception may occur should be placed inside:

Answer:  the try: branch



"""